README.md

```diff
@@ -78,14 +80,32 @@ vcs import src < rmf.repos
 ```
 
 Ensure all ROS 2 prerequisites are fulfilled,
+
```

Comment:
`wget https://raw.githubusercontent.com/open-rmf/rmf/main/rmf.repos` would get the latest source not 21.09, let's try to make everyhing here fixed to 21.09 and then add some sources compilation in the main readme, specifying that it currently supports Galactic.